docs/angular/guides/cli-overview.md

```diff
@@ -243,6 +243,35 @@ Any flags you pass to `run-many` that aren't Nx specific will be passed down to
 nx affected --target=build --prod
 ```
 
+## Loading Environment Variables
+
+By default, Nx will load any environment variables you place in the following files:
+
+1. `workspaceRoot/apps/my-app/.local.env`
+2. `workspaceRoot/apps/my-app/.env`
+3. `workspaceRoot/.local.env`
+4. `workspaceRoot/.env`
+
+Order is important. Nx will move through the above list, ignoring files it can't find, and loading environment variables into the current process for the ones it can find. If it finds an variables that's already been loaded into the process, it will ignore it. It does this for two reasons:
```

Comment:
```suggestion
Order is important. Nx will move through the above list, ignoring files it can't find, and loading environment variables into the current process for the ones it can find. If it finds any variables that have already been loaded into the process, it will ignore it. It does this for two reasons:
```